src/views/DatasetManager/components/DatasetList/DatasetList.js

```diff
@@ -32,8 +31,20 @@ export const DatasetList = () => {
   const { t } = useTranslation();
   const classes = useStyles();
   const [isRefreshConfirmationDialogOpen, setIsRefreshConfirmationDialogOpen] = useState(false);
-  const { sortedDatasetList, currentDataset, selectDataset } = useDatasetList();
+  const { sortedDatasetList, currentDataset, selectDataset, deleteDataset } = useDatasetList();
   const [displayedDatasetList, setDisplayedDatasetList] = useState(sortedDatasetList);
+  const selectedDatasetBeforeDelete = useRef(null);
+
+  useEffect(() => {
+    setDisplayedDatasetList(sortedDatasetList);
```

Comment:
Do we really need to update the `displayedDatasetList` state when the "currently selected dataset" changes?